src/main/java/core/basesyntax/MainApp.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class MainApp {
+    private static final int SIZE_FIGURE_ARRAY = 6;
+    private static final FigureSupplier figureSupplier = new FigureSupplier();
+
+    public static void main(String[] args) {
+        Figure[] figures = new Figure[SIZE_FIGURE_ARRAY];
+        for (int i = 0; i < figures.length; i++) {
+            if (i < 3) {
```

Comment:
It's better to reuse constants in expressions in order to make only one change if you require another number of figures:
```suggestion
            if (i < SIZE_FIGURE_ARRAY / 2) {
```